applications/models.py

```diff
@@ -239,6 +239,21 @@ def set_status(self, status, user=None):
     def get_status(self):
         return self.statuses.last()
 
+    def get_application_events_by_basket(self):
+        handled_application_events: [ApplicationEvent] = {}
+        for basket in sorted(
+            self.application_round_baskets.all(), key=lambda k: k.order_number
+        ):
+            handled = [
+                event
+                for events in handled_application_events.values()
```

Comment:
Is the return value of `.values()` needed here (it is queryset dict)? or would just `.all()` be good here (which is queryset with objects)?

Reply:
I'll check it out.

Reply:
With the current way it's done we need to use values, not a queryset dict as such, it's a dict with basket id as key and instances of events that belong to that basket as values. Maybe there is a better way to structure this?